src/main/java/Gronk.java

```diff
@@ -0,0 +1,174 @@
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+import java.util.Arrays;
+import java.util.ArrayList;
+
+import java.io.FileWriter;
+import java.io.File;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+
+public class Gronk {
+    public static String WELCOME = "\tHi, I'm Gronk!\n"
+            + "\tWhat are we up to today?";
+    public static String GOODBYE = "\tSystem closing. Goodbye!";
+    public static ArrayList<Task> tasks;
+
+    public static void lines() { // prints out a line
+        System.out.println("  ----------------------------------------");
+    }
+
+    public static void printMessage(String message) { // the system sends a message
+        lines();
+        System.out.println(message);
+        lines();
+    }
+
+    public static void printList() {
+        try {
+            if (tasks.size() == 0) {
+                throw new EmptyListException();
+            }
+            String m = "";
+            for (int j = 0; j < tasks.size(); j++) {
+                m += "\t" + Integer.toString(j + 1) + ". " + tasks.get(j).toString() + "\n";
+            }
+            printMessage(m.substring(0, m.length() - 1));
+        } catch (EmptyListException e) {
+            printMessage(e.toString());
+        }
+    }
+
+    public static void saveTasks() {
+        try {
+            FileWriter myWriter = new FileWriter("tasks.txt");
+            for (Task t: tasks) {
+                myWriter.write(t.saveFormat() + "\n");
+            }
+            myWriter.close();
+            printMessage("\tTasks saved successfully!");
+        } catch (IOException e) {
+            printMessage("\tFailed to save.");
+        }
+    }
+
+    public static void loadTasks() {
+        try {
+            File myFile = new File("tasks.txt");
+            System.out.println("\tWelcome back! Loading tasks...");
+            Scanner sc = new Scanner(myFile);
+            while (sc.hasNextLine()) {
+                loadMessage(sc.nextLine());
+            }
+        } catch (FileNotFoundException e) {
+            System.out.println("\tNo tasks saved. Initializing new session.");
+        }
+    }
+
+    public static void loadMessage(String m) {
+        String[] message = m.split(",");
```

Comment:
Plural form should be used on names representing a collection of objects. I am not sure about this, but seems like lists need to have plural forms.

Reply:
You're corrected, have resolved this in the latest round of refactoring. Thanks for the spot!